PROJECTS.yaml

```diff
@@ -0,0 +1,70 @@
+#
+# Summary about Commonhaus Foundation projects
+#
+# name: Project Name
+# logo: Link to preferred logo to use on commonhaus.org website
+# home: preferred landing page
+# repo: gitlab/github/... organization or project repository
+# wordmark: true if the above logo also contains the project name
+# description: a short-ish description of the project
+# link: website / URL for the project
```

Comment:
Do we need both `home` and `link` ?

Reply:
woops. no. link should have been removed/replaced by home (as distint from repo). Thanks